torchvision/transforms/functional.py

```diff
@@ -126,6 +126,9 @@ def to_tensor(pic):
 
     See :class:`~torchvision.transforms.ToTensor` for more details.
 
+    .. warning::
+        This method is deprecated, please use :meth:`~torchvision.transforms.functional.pil_to_tensor`.
+
```

Comment:
Are we truly ready to deprecate this or should we wait until the new Transforms API is finalize? Note that we have been discouraging people from using it by eliminating its use internally and removing it from the references in favour of `pil_to_tensor`. Unfortunately though that is not a drop-in replacement and you need 2 calls. One for `pil_to_tensor` and for `convert_image_dtype` and even then those are not directly equivalents.

When we do deprecate it, we should throw warnings. For now I would just remove this.